build/v1/types.go

```diff
@@ -1165,7 +1165,7 @@ type BuildRequest struct {
 
 	// triggeredBy describes which triggers started the most recent update to the
 	// build configuration and contains information about those triggers.
-	TriggeredBy []BuildTriggerCause `json:"triggeredBy" protobuf:"bytes,8,rep,name=triggeredBy"`
+	TriggeredBy []BuildTriggerCause `json:"triggeredBy,omitempty" protobuf:"bytes,8,rep,name=triggeredBy"`
```

Comment:
This should be safe

Reply:
Yeah, we'll have to hold that until it's fixed.  Agree it should be changed there (if no triggers were specified, set them, otherwise let the client set).

Reply:
https://github.com/openshift/openshift-apiserver/pull/93

Reply:
err and now that I have coffee I realize that this change here actually increases the odds of `TriggeredBy` being nil, because without the `omitempty` we would have serialized a non-nil zero length slice here.